tests/full-disk-encryption.spec.ts

```diff
@@ -14,11 +15,12 @@ test.describe('The main page', () => {
 
     test('Full-disk encryption', async ({ page }) => {
         await test.step("Select the product", async () => {
-            await page.getByLabel('SUSE ALP Server').check();
+            await page.getByLabel(process.env.PRODUCTNAME).check();
             await page.getByRole("button", { name: "Select" }).click();
         });
```

Comment:
After latest changes in the isos, we don't have product selection option for Dolomite.

Reply:
Updated.